addons/account/__init__.py

```diff
@@ -5,6 +5,7 @@
 from . import models
 from . import wizard
 from . import report
+from . import tests
```

Comment:
This goes against the general recommandation for test submodules in Odoo.
See `test_uninstall.py` for a better way.

Reply:
fair enough but note that there's a lot of other modules that do this already